spec.emu

```diff
@@ -385,12 +382,24 @@ location: https://github.com/tc39/proposal-measure/
     <p>It performs the following steps when called:</p>
     <emu-alg>
       1. Let _O_ be the *this* value.
-      1. Perform ? RequireInternalSlot(_O_, [[AmountData]]).
-      1. If _options_ is not an Object, throw a *TypeError* exception.
-      1. Let _v_ be _O_.[[AmountData]].
-      1. Let _s_ be Number::toString(_v_, 10).
-      1. Return ? Call (%Amount%, _O_, « _s_, _options_ »).
-      1. Return OrdinaryObjectCreate(*"%Amount.prototype%"*, « [[AmountData]], [[SignificantDigits]], [[FractionDigits]], [[Unit]] »).
+      1. Perform ? RequireInternalSlot(_O_, [[Value]]).
+      1. Let _options_ be ? GetOptionsObject(_options_).
+      1. If ? HasOwnProperty(_options_, *"currency"*), throw a *TypeError* exception.
+      1. If ? HasOwnProperty(_options_, *"unit"*), throw a *TypeError* exception.
+      1. Let _fractionDigits_ be ? GetOption(_options_, *"fractionDigits"*, ~number~, ~empty~, *undefined*).
+      1. Let _significantDigits_ be ? GetOption(_options_, *"significantDigits"*, ~number~, ~empty~, *undefined*).
+      1. If both _fractionDigits_ and _significantDigits_ are not *undefined*, throw a *RangeError* exception.
+      1. If ℝ(_significantDigits_) is not an integer, throw a *RangeError* exception.
+      1. If ℝ(_significantDigits_) < 1, throw a *RangeError* exception.
+      1. If ℝ(_fractionDigits_) is not an integer, throw a *RangeError* exception.
+      1. If ℝ(_fractionDigits_) < 0, throw a *RangeError* exception.
```

Comment:
This set of rules will always throw an error for any combination of _fractionDigits_ and _significantDigits_.

Reply:
I think that's what I had in mind? What I'm trying to express is that we can't have both. And if either one is specified, they need to be sensible values.

It's possible that you could provide both but in that case I'd like to check that they're coherent. For instance, if we start with `"1.23`, we could do `fractionDigits: 2` and `significantDigits: 3`, and that's OK, but that requires some checking. Is that what you had in mind?